src/components/schedule-inputs.tsx

```diff
@@ -23,6 +25,16 @@ export function ScheduleInputs(props: ScheduleInputsProps): JSX.Element | null {
 
   const timezoneLabel = trans.__('Time zone');
 
+  let cronString;
+  try {
+    if (props.schedule !== undefined && !props.errors['schedule']) {
+      cronString = cronstrue.toString(props.schedule);
+    }
+  }
+  catch (e) {
+    // Do nothing; let the errors or nothing display instead
```

Comment:
Why not console error this?

Reply:
Because it's not an error; I have to have the `try`/`catch` block because, if a cron expression is not valid (and `''` is not valid), `cronstrue` throws an error that breaks the entire page, when really we need to display the error in `props.errors` or nothing at all (e.g., for initial load)

Reply:
I like the idea of having only one dependency; I can see after our 10:00 meeting whether I can use `cronstrue` to capture errors.

`cron` has a lot of edge cases; for example, `0 5 30 2 *` (at 05:00 every year on February 30) technically passes regexes, but because that date is impossible, a job with this schedule will never run.